### ISSUE TITLE
helmfile sync hangs when 'installed: false' is set and the release does not exist.

### ISSUE THREAD

**Issue by jim-barber-he** [roboll/helmfile#423]
Hi.

I've encountered an issue where I am trying to manage our releases using the `installed:` field in `helmfile.yaml`

If I set `installed: false` when the release has never been installed before and run `helmfile sync` it will show an error saying the release does not exist,  and then never return the prompt.

An minimal `helmfile.yaml` file that deploys a busybox release that just runs `tail -f /dev/null` to  demonstrate the problem is as follows:
```
---
repositories:

  - name: jfelten
    url: https://jfelten.github.io/helm-charts/charts

releases:

  - name: busybox
    chart: jfelten/busybox
    installed: false
```

When run, it produces the following error:
```
$ helmfile sync
Adding repo jfelten https://jfelten.github.io/helm-charts/charts
"jfelten" has been added to your repositories

Updating repo
Hang tight while we grab the latest from your chart repositories...
...Skip local chart repository
...Successfully got an update from the "jfelten" chart repository
...Successfully got an update from the "incubator" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈ Happy Helming!⎈ 

Getting status busybox
Error: getting deployed release "busybox": release: "busybox" not found
```

At this point the prompt never returns.

Note that if I use `helmfile apply` the same error is output, but the prompt is returned with exit code 0, presumably because the behind the scenes diff that is run finds no change and so the helm update is never run.

**Comment by pdemagny** [comment]
I can confirm this behavior, with a local chart, but i guess it doesn't matter.
The use-case is that i wanted to include "optional" charts in my helmfile by putting a `installed: false` on them that users could switch to `installed: true` later on.

**Comment by mumoshu** [comment]
@jim-barber-he @pdemagny Hey! Thanks for reporting. I've finally got a chance to fix this. The fix will be included in the upcoming v0.45.1.

**Comment by pdemagny** [comment]
Awesome, thanks a lot !
